sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Common/tests/Microsoft.Azure.WebJobs.Extensions.Storage.Common.Tests.csproj

```diff
@@ -4,7 +4,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.WebJobs" />
+    <PackageReference Include="Microsoft.Azure.WebJobs" VersionOverride="3.0.36" />
```

Comment:
Same here

Reply:
Updated this value globally in `eng/Packages.Data.props`